tensorrt_llm/_torch/models/modeling_qwen2vl.py

```
@@ -458,6 +473,11 @@ def __init__(
         **kwargs,
     ) -> None:
         model_config.pretrained_config.rope_scaling['type'] = 'mrope'
+
+        # NOTE: Setting disable_fuse_rope to True to do mrope fusion in the model engine by pre-computing rotary_cos_sin in the model engine
```

Ruff (0.12.2)

Line too long (140 > 120)

(E501)